src/app/(nousConnaitre)/dateCles/page.js

```
@@ -1,205 +1,225 @@
+        <div className="relative ml-[8rem]">
+          <img
+            src="/dateCles/site bab rayan dates clés-02.png"
+            alt="element"
+            className="w-[40rem] h-[20rem]"
+          />
+        </div>
+
+        {/* Texte sous l'image à gauche */}
+        <div className="relative grid place-items-start  ml-[22rem]">
+          <ul className="list-disc list-inside text-left">
+            <li className="pl-[1rem]">Création de l'association Bab Rayan</li>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)